src/store/zap/actions.js

```diff
@@ -14,803 +14,820 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-import Vue from 'vue'
```

Comment:
The whole file seem to have changed and makes it very difficult to actually review the changes.
Can you fix this?

Reply:
Fixed